lib/start.js

```diff
@@ -32,6 +32,10 @@ function start(_opts, _cb) {
     opts.spawnCb = noop;
   }
 
+  if (opts.version.startsWith('4.') && opts.seleniumArgs.indexOf('standalone') === -1) {
```

Comment:
I forgot to take into account that we should only add standalone if the mode is not specified explicitly.
https://www.automatetheplanet.com/selenium-webdriver-4-overview/
```
// add somewhere to the top
const seleniumModes = ['standalone', 'hub', 'node']

// replace opts.seleniumArgs.indexOf('standalone') === -1 with
!seleniumModes.some((a) => seleniumArgs.includes(a))
```

Reply:
Hm, I think it would be even better if at least one arg passed and that's it. 
```suggestion
  if (opts.version.startsWith('4.') && opts.seleniumArgs.length === 0) {
```